### PR TITLE
Release notes for v1.2.1 decrediton

### PR DESCRIPTION
**Pull request by alexlyp**

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+
+See  manifest-decrediton-v1.2.1.txt, and the package specific manifest files for sha256 sums and the associated .asc files to confirm those shas.
+
+See [README.md](./README.md#verifying-binaries) for more info on verifying the files.
+
+
+# decrediton v1.2.1
+
+This decrediton patch release covers a few commonly reported issues shortly
+after v1.2.1 was released.  Most importantly this fixes an issue causing
```
Comment (raedah) [inline release-notes.md]: This decrediton patch releases covers a few issues that were commonly reported shortly after v1.2.0 was released.

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+* Fix balance showing 0 after ticket/votes calculated for maturity.  
+
+* Seed pasting was not showing errors and would just blank out the words.  Now
+  an error is shown when that occurs.
+
+* Fix various PT-Br translation errors.
+
+* Due to dcrd v1.2.0 database upgrade, users would be shown "Waiting for RPC..." 
+  for a very long time with no other information.  Now the first lines of both
+  dcrd and dcrwallet logs are show to hopefully give the user more information
```
Comment (davecgh) [inline release-notes.md]: are shown

**Comment by infertux** [comment]
Shouldn't the `release-notes.md` file be deleted altogether since it's out-of-date and the actual release notes are available at https://github.com/decred/decred-binaries/releases? I think it's better to have a single location for the release notes as it's less confusing for people looking for them and requires less maintenance to keep all the copies in sync. Happy to open a PR to delete the file.

**Comment by alexlyp** [comment]
Yah we can discuss this further

**Pull request merged**